ProcessLib/SourceTerms/VolumetricSourceTermData.h

```diff
@@ -0,0 +1,39 @@
+/**
+ * \copyright
+ * Copyright (c) 2012-2018, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ *
+ */
+
+#pragma once
+
+namespace ProcessLib
+{
+
+template <typename T>
+struct Parameter;
+
+struct VolumetricSourceTermData final
+{
+    explicit VolumetricSourceTermData(
+        Parameter<double> const& volumetric_source_term_)
+        : volumetric_source_term(volumetric_source_term_)
+    {}
+
+    VolumetricSourceTermData(VolumetricSourceTermData&&) = default;
+
+    //! Copies are forbidden.
+    VolumetricSourceTermData(VolumetricSourceTermData const&) = delete;
```

Comment:
Question: If the base class has a deleted assignment op or ctor, is a default assignment op or ctor created implicitly in the derived class?